### ISSUE TITLE
[request] management of "read" issues / volumes

### ISSUE THREAD

**Issue by mnbf9rca** [evilhero/mylar#1097]
I would like to be able to "archive" comics, volumes or issues that i've read so that they don't clutter up a) my dropbox (which is how i get them to comicat), and b) the user interface. It would be great if this functionality:
- hid the comics from the UI with an option to reinstate them,
- moved the comic folder to another folder
- had the option of continuing to receive updates or not (as there are some comics which i don't want to read while a story arc develops so that i can read it all in one go).

**Comment by evilhero** [comment]
Since the Read Management section is still in the process of being written , some of these are viable options that I might be able to integrate at some point.

Having an additional status for archived read issues and then being able to toggle the view of said issues in the reading tab. 

Moving to an additional folder is also possible (mylar already has a secondary series directory it will watch and monitor for issues but will not write new issues into it).

However the continuing updates might be problematic as it relies on the actual story arcs to process / update and adding in the ability to 'ignore' certain issues would be a pretty big rewrite for a small gain.

Keep in mind that all of the above is based on the fact that this is in dealing with the Reading List Management tab, not from the series detail page (hiding specific issues from the UI would never happen in that context)

**Comment by mnbf9rca** [comment]
#3 is not saying "only download some issues", but "keep downloading them while the comic is archived" i.e. keep this comic updated in the background.

RE: the display - perhaps give people the option to have the Reading List Management tab as the default "view" would help - i've got abotu 40-50 comics in my list and it's getting hard to manage now. Is there any documentation on the "reading list management"? Perhaps it would be good to have a folder as the destination, instead of using SFTP? I use dropbox + comicat to sync issues to my tablet.

**Comment by mnbf9rca** [comment]
also as far as i can tell, the reading list page is really about story arcs, especially those which cross comics.

**Comment by evilhero** [comment]
TBH, the reading list is about tracking issues that a user wants to read/have read. It doesn't have one preference vs the other (My reading list accumulates due to the new weekly pull's getting automatically added to the reading list that have 0 to do with any story arcs I might be tracking at the time). The benefit of using the reading list is that it will track the story arcs in a round-about way, but in reality, it has no linkage to the Story Arc's aside from said arc issues being present in the reading list to be tracked/sent to a device.

While I can understand some of the request, the 'keep downloading them while the comic is archived' is abit confusing (maybe it's cause I only had one cup of java..I dunno). Issues are archived, not comics (in the sense that comics are parents, and issues are the children) - regardless of whether it's story arcs/series. It sounds like what you're wanting is the addition of another type of status that would fit somewhere prior to  'Added' and well before  'Downloaded'(kind of like 'Queued'), wherein items sit until you force them into an 'Added' status (which therein gets synced up afterwards and the status would change to 'Downloaded' thereafter).

I'm not sure what the purpose of making the Reading List tab as the default 'view' would accomplish (aside from more work for me, but that aside), or probably more to the point - not sure exactly what it is that you're wanting to change the default from. The Reading List is it's own page within Mylar now (it has been that way for awhile), but changing out the main view (?) for a non-default view brings with it other problems as well - I'm currently trying to figure out how to make the Reading List / Story Arc more prominent in the UI than having it buried in the Manage tab. 

Adding a folder to drop reading issues that are in an 'Added' status is coming - yes. The SFTP/Android option was primarily for my own usage as that's what I was interested in trying to do ;)

**Comment by mnbf9rca** [comment]
i dont think it's your lack of caffeine - i'm just not explaining it clearly. Let me try again. I dont know whether "reading list" is the right place for these or not.

* I have a lot of comics. The "home" page, with a single, long list which spans 3 "paginations" is hard to keep track of things on.
* Some of the comics are "continuing" and some are "ended".
* For the ones which have ended, it's annoying to have them clutter up the screen
* for the ones which are continuing, if there's an interesting story going on (like "Zero Year" in Batman), i prefer to wait until the entire story has been published so that i can read it in one go (I'm impatient).

So, comics (for me) have 3 "status":
* (continuing or ended) and reading - these ones i want to sync via a specific folder in my dropbox AND keep a copy in my main archive (so i can delete from dropbox when read without worrying about losing the issue)
* (ended and archived) or (continuing and "stopped")- these ones i want to move elsewhere. I'm not reading them.
* continuing and waiting - I am reading these but i'm waiting for a story to pan out. I dont mind where these are, but i dont want them in my "currently being read" folder as then they'll show up in ComiCat and i'll get frustrated :) - this is probably just me though, and really the above 2 classes are (I think) most likely to reflect what most people have. So this is a "super nice to have".

there are also some other improvements i could think of e.g. remember sort order, allow thumbnail/bookshelf views, allow logical grouping of e.g. by character, add additional status columns (like "archive") and allow to sort/filter on these, but perhaps one way to do it would be to display them in the same structure that the user has them stored on disk, and to allow us to move them between this structure (e.g. have an "archive" folder as a system setting etc.). So if i want to display them based on publisher or by year or character or whatever, as long as i can define a rule for it in the renamer (so mylar knows where to put new comics i guess), i can use that, and the listing simply reflects that storage, with the addition of 2 "high level stores" ("archive" and "active" or something).

Does that help?
